config_converter/config_mapper/config_mapper.py

```diff
@@ -76,19 +79,20 @@ def extract_root_dirs(config_obj: config_pb2.Directive) -> tuple:
         if d.name not in plugin_prefix_map:
             stats['entities_skipped'] += 1
             stats['attributes_skipped'] += _get_aggregated_num_attributes(d)
-            print(f'Currently we do not support plugins of {d.name}')
+            logging.warning('Currently we don\'t support plugins of %s',
```

Comment:
```suggestion
            logging.warning('Currently we don\'t support mapping plugin %s, as the functionality of %s is not supported in master agent yet',
```

This may be lengthy, but just want to let the users know why this tool doesn't support mapping plugin X. Feel free to use your own word to achieve the goal.

Reply:
Will do!